extensions/azure/processors/PutAzureBlobStorage.cpp

```diff
@@ -90,14 +81,13 @@ const core::Relationship PutAzureBlobStorage::Failure("failure", "Unsuccessful o
 
 void PutAzureBlobStorage::initialize() {
   // Set the supported properties
-  setSupportedProperties({
+  updateSupportedProperties({
```

Comment:
Why not set? 
Update would make sense to me in case the base class would already set some properties, but that doesn't seem to happen as far as I see.

Reply:
The base class `AzureStorageProcessorBase` does set the `AzureStorageCredentialsService` property beforehand in its constructor.

Reply:
Updated in f02f86c